### PR TITLE
polymc: update to 1.4.0

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

Update `polymc` to 1.4.0

Package(s) Affected
-------------------

`polymc` 1.4.0

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by OriginCode** [comment]
LGTM on amd64:
圖片

File: extra-games/polymc/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,11 @@
+abinfo "Building polymc ..."
+mkdir -pv "$BLDDIR"
+cd "$BLDDIR"
+cmake "$SRCDIR" -DCMAKE_BUILD_TYPE=Release \
```
Comment (cthbleachbit) [inline extra-games/polymc/autobuild/build]: Nit: Include `${CMAKE_DEF}` when invoking cmake from custom build scripts.

Comment (cthbleachbit) [reply extra-games/polymc/autobuild/build]: They define `CMAKE_BUILD_TYPE` and `CMAKE_INSTALL_PREFIX` so you don't have to

Comment (MingcongBai) [reply extra-games/polymc/autobuild/build]: > Nit: Include `${CMAKE_DEF}` when invoking cmake from custom build scripts.

Yeah, and `Release` is wrong, you need `RelWithDebInfo`. But you should use CMAKE_DEF, as @cthbleachbit suggests.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — DISMISSED** [review]

**Comment by OriginCode** [comment]
Issue found
圖片

**Comment by OriginCode** [comment]
The issue above is fixed.
圖片

**Review by MingcongBai — DISMISSED** [review]

**Review by cthbleachbit — APPROVED** [review]
it werks (TM)

**Pull request merged**